tasks/nfs-server.yml

```diff
@@ -14,18 +14,18 @@
       filesystem:
         fstype: "{{ nfs_fstype }}"
         dev: "{{ nfs_disk_location }}"
+        resizefs: true
```

Comment:
Thanks, looks good. One question though.. what happens to the filesystems that need to be mounted - is it a noop? or does it result in a failure?

Reply:
sorry I missed to reply this one. Right now I tried this playbook again and running the `mount` module two times is not a problem. It's a noop and the playbook can be executed as many times as needed without issues and without breaking idempotence (using command module it's executed on every run)